source/Octopus.Tentacle.CommonTestUtils/Builders/ScriptBuilder.cs

```diff
@@ -41,7 +41,7 @@ public ScriptBuilder CreateFile(string file)
 touch '{file}'
 ");
             windowsScript.AppendLine($@"
-New-Item '{file}'
+New-Item -type file '{file}'
```

Comment:
In recent versions of Powershell, the `type` arg is optional and a sensible default is chosen by Powershell.

However, in _older_ versions of Powershell, like on Windows Server 2012, omitting the arg brings up a user prompt to specify the value. Obviously, we can't do this in Tentacle (as Powershell is in non-interactive mode), so Tentacle was failing here.

In the future, it would be helpful for test failures to log the Tentacle output, which would've made finding this one a lot easier.